docs/releases/4.1.txt

```diff
@@ -246,6 +246,9 @@ backends.
 * ``BaseDatabaseFeatures.has_case_insensitive_like`` is changed from ``True``
   to ``False`` to reflect the behavior of most databases.
 
+* ``BaseDatabaseIntrospection.get_key_columns()`` is removed, use
+  ``BaseDatabaseIntrospection.get_relations()`` instead.
```

Comment:
Perhaps these should just be "DatabaseIntrospection". The "Base" implementations can't be used anyway.